src/test/java/gameboy/gamepack/racinggame/data/entity/DriverTest.java

```diff
@@ -0,0 +1,21 @@
+package gameboy.gamepack.racinggame.data.entity;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class DriverTest {
+
+    @Test
+    @DisplayName("drive 정상 테스트")
+    void drive() {
+        //given
+        Driver driver = new Driver();
+        //when
+        int result = driver.drive();
+        //then
+        assertThat(result).isBetween(0, 1);
```

Comment:
drive는 동사라서 상태를 변경하는 행위로 보입니다.
자동차를 이동시킨 후 자동차의 위치를 꺼내서 비교하는 게 좀더 자연스러워 보이는데 어떻게 생각하시나요?

```suggestion
        Driver driver = new Driver();
        //when
        driver.drive();
        //then
        assertThat(driver.position()).isBetween(0, 1);
```

Reply:
자동차 위치가 0인지 1인지 제어가 되지 않아 between으로 테스트 케이스를 만들었네요
아마 랜덤 때문에 이런 코드가 나온 것 같아요

뒷부분에서도 리뷰 드리겠지만
랜덤 같이 제어할 수 없는 부분을 어떻게 테스트하면 좋을지 한번 고민해 보셨으면 좋겠습니다.